src/ci/github-actions/ci.yml

```diff
@@ -261,7 +262,7 @@ jobs:
     name: try
     env:
       <<: [*shared-ci-variables, *prod-variables]
-    if: github.event_name == 'push' && github.ref == 'refs/heads/try' && github.repository == 'rust-lang-ci/rust'
+    if: github.event_name == 'push' && github.ref == 'refs/heads/try' && github.ref != 'refs/heads/try-perf' && github.repository == 'rust-lang-ci/rust'
```

Comment:
Why did you add this `if` condition?

`github.ref != 'refs/heads/try-perf'` is always going to be true if `github.ref == 'refs/heads/try'` is also true.